samples/apps/copilot-chat-app/webapp/src/components/chat/ChatResourceList.tsx

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import {
```

Comment:
In general, the design with Ruokan's Figma
https://www.figma.com/file/9U23g6fAARTvJMSB2drvHU/SK_Bot_Interactions?type=design&node-id=917-62001&t=KUz1fPRA392S1ON1-0

(Unless you're referencing another design)

Reply:
correct, here's a screenshot of the design:
<img width="654" alt="image" src="https://github.com/microsoft/semantic-kernel/assets/52973358/cbc3f016-36a8-413b-89ea-18f464c6b667">

looks like sorting isn't currently implemented so I can add that in.

Reply:
done